### ISSUE TITLE
Question regarding Table 4 results

### ISSUE THREAD

**Issue by YUE-FAN** [huangjk97/T2T#4]
Dear Authors,

I have a question regarding the results in Table 4 in your paper. T2T gives 99.85% AUROC on CIFAR-ID-50 with TIN. However, this setting does not have a hold-out test set for OOD classes because you use the test set of TIN for training. So on which test set do you compute the result 99.85%?

Using your code, I can only reproduce this number if evaluating the model on the training set (CIFAR-ID-50 unlabeled set + TIN). So my understanding is this number is training AUROC and should not be compared with other numbers.

**Comment by buzhiqimeiliuqiangdong** [comment]
> 
兄你有这篇论文测试AUROC的代码吗？在作者的代码里好像没给。

**Comment by LxxxxK** [comment]
> Dear Authors,
> 
> I have a question regarding the results in Table 4 in your paper. T2T gives 99.85% AUROC on CIFAR-ID-50 with TIN. However, this setting does not have a hold-out test set for OOD classes because you use the test set of TIN for training. So on which test set do you compute the result 99.85%?
> 
> Using your code, I can only reproduce this number if evaluating the model on the training set (CIFAR-ID-50 unlabeled set + TIN). So my understanding is this number is training AUROC and should not be compared with other numbers.

兄你有这篇论文里测试AUROC的代码吗，作者的代码里好像没给
